pkgs/by-name/pm/pmtiles/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "pmtiles";
+  version = "1.11.3";
```

Comment:
optional: 1.12.0 is out, I'll leave it up to you to decide if you want to do part of this PR or subsequent PR